includes/Admin/AdminBar.php

```diff
@@ -91,6 +91,38 @@ public function dokan_admin_toolbar() {
         do_action( 'dokan_render_admin_toolbar', $wp_admin_bar );
     }
 
+    /**
+     * Get admin menus data for dokan.
+     *
+     * @since DOKAN_LITE_SINCE
```

Comment:
DOKAN_LITE_SINCE won't work here. Kindly replace with the upcoming release version.